fbw-a380x/src/systems/instruments/src/AtcMailbox/AtcMailbox.tsx

```diff
@@ -0,0 +1,61 @@
+//  Copyright (c) 2024 FlyByWire Simulations
+//  SPDX-License-Identifier: GPL-3.0
+
+import { DisplayComponent, EventBus, FSComponent, Subject, VNode } from '@microsoft/msfs-sdk';
+import { Button } from 'instruments/src/MFD/pages/common/Button';
+import { MouseCursor } from 'instruments/src/MFD/pages/common/MouseCursor';
+import { CdsDisplayUnit, DisplayUnitID } from '../MsfsAvionicsCommon/CdsDisplayUnit';
+
+import './style.scss';
+
+export interface AtcMailboxProps {
+  readonly bus: EventBus;
+}
+
+export class AtcMailbox extends DisplayComponent<AtcMailboxProps> {
+  private readonly topRef = FSComponent.createRef<HTMLDivElement>();
+
+  private readonly mouseCursorRef = FSComponent.createRef<MouseCursor>();
+
+  private onMouseMove(ev: MouseEvent) {
+    this.mouseCursorRef.getOrDefault()?.updatePosition(ev.clientX, ev.clientY - 768);
+  }
+
+  private onMouseMoveHandler = this.onMouseMove.bind(this);
+
+  public onAfterRender(node: VNode): void {
+    super.onAfterRender(node);
+
+    this.topRef.instance.addEventListener('mousemove', this.onMouseMoveHandler);
+  }
+
+  destroy(): void {
+    this.topRef.getOrDefault()?.removeEventListener('mousemove', this.onMouseMoveHandler);
+
+    super.destroy();
+  }
+
+  render(): VNode | null {
+    return (
+      <CdsDisplayUnit bus={this.props.bus} displayUnitId={DisplayUnitID.Sd}>
```

Comment:
Hmmm, does this not conflict with the displayunit component of the SD?

Reply:
In principle yes, but it always shows the same on top of each other so it isn’t an issue. In principle we need a CDS DU for React not showing the overlays, just implementing the power/on/off/dim states

Reply:
@Benjozork I now disabled the Cds stuff from the underlying React component, only the overlay from the avionics framework is used. I'd appreciate a quick review of that commit, then I can merge